scss/tablet.scss

```diff
@@ -0,0 +1,224 @@
+@import "vars";
+
+@media (min-width: 768px) {
+  .hamburger-container {
+    margin-right: 50px;
```

Comment:
such indent must be set by parent , I was told abiut parent padding etc. on the meeting

Reply:
done